django/db/backends/sqlite3/base.py

```diff
@@ -242,6 +243,7 @@ def get_new_connection(self, conn_params):
         create_deterministic_function('PI', 0, lambda: math.pi)
         create_deterministic_function('POWER', 2, none_guard(operator.pow))
         create_deterministic_function('RADIANS', 1, none_guard(math.radians))
+        create_deterministic_function('RAND', 0, random.random)  # Don't override built-in RANDOM().
```

Comment:
`RAND()` is non-deterministic. I fixed this.

Reply:
So it is! Thanks.